src/main/java/com/skhedul/skhedulbe/rest/UserController.java

```
@@ -31,4 +27,10 @@
     UserDTO savedUser = userService.saveUser(user);
     return new ResponseEntity<>(savedUser, HttpStatus.CREATED);
   }
+
+  @DeleteMapping(path="/{name}")
+  public ResponseEntity<Void> deleteUser(@PathVariable("name") final String userName){
```

We may want to return an error body for some cases in the future. So, we can use wildcards here (ResponseEntity<?>).
 more on wildcards, https://www.geeksforgeeks.org/wildcards-in-java/
Be aware that SonarCube gets angry when it sees wildcards in the method signature, but we can screw that for now or suppress that if we ever use it.